src/sentry/static/sentry/app/views/settings/organizationRelays/relays.tsx

```diff
@@ -34,154 +37,223 @@ type State = AsyncComponent['state'] & {
+              link: <ExternalLink href={RELAY_DOCS_LINK} />,
+            }
+          )}
+        </TextBlock>
+        <StyledPanelTable
+          isEmpty={relays.length === 0}
+          emptyMessage={t('No relays keys have been added yet.')}
+          headers={[t('Display Name'), t('Relay Key'), t('Date Created'), '']}
+        >
+          {relays.map(({publicKey: key, name, created, description}) => {
+            const maskedKey = '*************************';
+            return (
+              <React.Fragment key={key}>
+                {description ? (
+                  <Name>
+                    <Text>{name}</Text>
+                    <QuestionTooltip position="top" size="sm" title={description} />
+                  </Name>
+                ) : (
+                  <Text>{name}</Text>
+                )}
+                <KeyWrapper>
+                  <Key content={maskedKey}>{maskedKey}</Key>
+                  <IconWrapper>
+                    <Clipboard value={key}>
+                      <Tooltip title={t('Click to copy')} containerDisplayMode="flex">
+                        <IconCopy color="gray500" />
+                      </Tooltip>
+                    </Clipboard>
+                  </IconWrapper>
+                </KeyWrapper>
+                <Text>
+                  {!defined(created) ? t('Unknown') : <DateTime date={created} />}
+                </Text>
+                <Actions>
+                  <StyledButton
+                    title={t('Edit Key')}
+                    label={t('Edit Key')}
+                    icon={<IconEdit />}
+                    onClick={this.handleOpenEditDialog(key)}
+                  />
+                  <StyledButton
+                    title={t('Delete Key')}
+                    label={t('Delete Key')}
+                    onClick={this.handleDelete(key)}
                     icon={<IconDelete />}
                   />
-                </Content>
-              )
-            )}
-          </PanelBody>
-          <PanelAction>
-            <Button href={RELAY_DOCS_LINK} target="_blank">
-              {t('Read the docs')}
-            </Button>
-            <Button onClick={this.handleAdd} priority="primary">
-              {t('Add Relay')}
-            </Button>
-          </PanelAction>
-        </Panel>
+                </Actions>
+              </React.Fragment>
+            );
+          })}
+        </StyledPanelTable>
       </React.Fragment>
     );
   }
 }
 
 export default Relays;
 
-const Content = styled('div')`
-  display: grid;
-  grid-template-columns: 1fr max-content;
-  align-items: center;
-  border-bottom: 1px solid ${p => p.theme.borderDark};
-  padding: ${space(1)} ${space(2)};
-  :last-child {
-    border-bottom: 0;
+const StyledPanelTable = styled(PanelTable)`
+  grid-template-columns: repeat(3, auto) max-content;
+  > * {
+    padding: ${space(1)};
+    @media (min-width: ${theme.breakpoints[0]}) {
+      padding: ${space(1)} ${space(2)};
+    }
```

Comment:
I had changed the padding because of the size of the button, it seemed to have a lot of space..but now it doesn't make sense. I've updated the code ;-)